src/libstd/io/cursor.rs

```diff
@@ -391,12 +391,14 @@ impl Write for Cursor<Box<[u8]>> {
     fn flush(&mut self) -> io::Result<()> { Ok(()) }
 }
 
+#[unstable(feature = "cursor_partial_eq", issue = "0")]
```

Comment:
Trait implementations are always stable.

Reply:
okay so why did I get an error in rust-highfive? should I remove the last commit?

Reply:
You should add a `#[stable]` attribute with the version it's expected to release in -- probably 1.42 by the time fcp concludes, since 1.41 will branch to beta early next week.